mongo/integration/client_side_encryption_prose_test.go

```diff
@@ -1883,7 +1886,10 @@ func TestClientSideEncryptionProse(t *testing.T) {
 					}),
 			}
 
-			wcMajority := writeconcern.New(writeconcern.WMajority(), writeconcern.WTimeout(1*time.Second))
+			wcMajority := &writeconcern.WriteConcern{
```

Comment:
Can we use the convenience function in the example, instead of using the "majority" magic string? 

```go
wcMajority := writeconcern.Majority()
wcMajority.WTimeout = 1 * time.Second
```